chargedUp/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -91,18 +77,47 @@ public void drivetrainDefaultCommand(){
    * joysticks}.
    */
   private void configureBindings() {
-    //Spins Motor if April Tags are Recognized for 20 Ticks
-    m_driverController.a().
-        onTrue(new targetFinding(m_drivetrainSubsystem, m_visionSubsystem));
+        m_auxController.start().
+        onTrue(new InstantCommand(()-> m_armSubsystem.spinTelescopingMotor(0.8)))
+        .onFalse(new InstantCommand(()-> m_armSubsystem.stopTelescopingMotor()));
 
-    m_driverController.b().whileTrue(new InstantCommand(()-> m_drivetrainSubsystem.resetPosition()));
-  }
+        m_auxController.back().
+        onTrue(new InstantCommand(()-> m_armSubsystem.spinTelescopingMotor(-0.8)))
+        .onFalse(new InstantCommand(()-> m_armSubsystem.stopTelescopingMotor()));
+
+        //This runs Endeffector to Collect Cube
+        m_auxController.b().
+        onTrue(new InstantCommand(()-> m_armSubsystem.spinEndEffector(0.5)))
+        .onFalse(new InstantCommand(()-> m_armSubsystem.spinEndEffector(0.07)));
 
+        //This runs Endeffector to Collect Cone
+        m_auxController.a().
+        onTrue(new InstantCommand(()-> m_armSubsystem.spinEndEffector(0.5)))
+        .onFalse(new InstantCommand(()-> m_armSubsystem.stopEndEffector()));
+        
+        
+        // This runs Endeffector to eject game peices
+        m_auxController.rightTrigger().
+        onTrue(new InstantCommand(()-> m_armSubsystem.spinEndEffector(-0.2)))
+        .onFalse(new InstantCommand(()-> m_armSubsystem.stopEndEffector()));
+  
+        m_auxController.y().
```

Comment:
Can we add a comment saying what this is intended to do

Reply:
👍